modules/govuk_jenkins/manifests/jobs/ask_export.pp

```diff
@@ -69,14 +69,17 @@
     notify  => Exec['jenkins_jobs_update'];
   }
 
-  if $run_daily {
-    @@icinga::passive_check { "ask_export_${::hostname}":
-        ensure              => present,
-        service_description => $service_description,
-        host_name           => $::fqdn,
-        freshness_threshold => 86400, # 24 Hours
-        action_url          => "https://${deploy_jenkins_domain}/job/ask-export/",
-        # notes_url           => monitoring_docs_url(ask-export);
-    }
+  $freshness_threshold = $run_daily ? {
+    true  => 86400, # 24 hours
+    false => '',
+  }
+
+  @@icinga::passive_check { "ask_export_${::hostname}":
```

Comment:
Is there a functional difference between not running this block at all and setting the freshness threshold to an empty string?

Reply:
Yeah - this sets up the check in integration and it'll show green if the job succeeds and red if the job fails, but has no check for freshness.

Reply:
nice one, thanks :)